ui/pages/first-time-flow/seed-phrase/confirm-seed-phrase/index.scss

```diff
@@ -85,7 +85,7 @@
       display: inline-flex;
 
       &:hover {
-        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
+        box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
       }
     }
 
```

Comment:
This isn't used anymore. What this looks like:
<img width="814" alt="Screen Shot 2022-07-19 at 3 28 23 PM" src="https://user-images.githubusercontent.com/8112138/179859178-62a1d1f3-4f3a-4e99-88d2-329ceabef2b6.png">

Updated UI
<img width="648" alt="Screen Shot 2022-07-19 at 3 29 10 PM" src="https://user-images.githubusercontent.com/8112138/179859182-a5252c4e-4e56-485d-bea5-88d7caef15d9.png">